src/server/Document/component.test.jsx

```diff
@@ -17,10 +17,10 @@ describe('Document Component', () => {
       <script src="igbo.js" />
     </>
   );
-  const styleTags = (
-    <style data-styled-components="abc">{'html { color: red; }'}</style>
-  );
-  const styleTagsAmp = <style amp-custom="">{'html { color: red; }'}</style>;
+  // const styleTags = (
```

Comment:
Are these comments supposed to be here still?

Reply:
No, these have been long-pruned!